go-controller/pkg/ovn/secondary_layer3_network_controller.go

```diff
@@ -747,6 +751,30 @@ func (oc *SecondaryLayer3NetworkController) addUpdateRemoteNodeEvent(node *kapi.
 	return err
 }
 
+// addNodeSubnetEgressSNAT adds the SNAT on each node's switch in L3 networks
```

Comment:
isnt the SNAT added to the cluster router?

Reply:
good catch